### PR TITLE
Add test for user permissions

### PR DESCRIPTION
**Pull request by kgieseking**
Added some tests for user permissions and user auth, changed a few step definitions to reflect these tests.

### CODE REVIEW

File: features/step_definitions/user_auth_steps.rb [hunk 1]
```diff
@@ -14,3 +14,12 @@
 Given /^I have an account for (.*)$/ do |account|
   User.create!(email: account, password: 'hunter2')
 end
+
+<<<<<<< HEAD
```
Comment (susannasouv) [inline features/step_definitions/user_auth_steps.rb]: This needs to be removed

File: features/user_permissions.feature [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Scenario: Nobody but the Events Manager should be able to add events
+  Given I am not a Events Manager
+  And I am on the Upcoming Events page
+  Then I should not see the Add Event button
+
+Scenario: Accounts Manager has permission to change other's accounts permissions
+  Given I am a Accounts Manager
+  And I am on the home page
+  Then I should see Manage Accounts in the navbar 
+<<<<<<< HEAD
```
Comment (susannasouv) [inline features/user_permissions.feature]: This should also be removed

**Review by susannasouv — CHANGES_REQUESTED** [review]
Remove the [file_name].swp files; that's a sign that you were in middle of editing those files elsewhere and didn't save so you might want to check on those...

**Comment by jeremyjpark** [comment]
You might also want to clean up your commit history so that in the end it looks more concise. I can work on the rebasing with you.

**Review by susannasouv — APPROVED** [review]

**Review by aganpule — APPROVED** [review]

**Pull request merged**